test/protractor/color-validator.spec.js

```diff
@@ -12,67 +12,120 @@ describe('Directive: Color Validator', function() {
 
```

Comment:
I accidentally committed this change when trying to retrigger the build using --allow-empty.. Just rolling with it. sorry.

Reply:
lol just kidding, it's fine